test/imp/sum-breadth-limit-three-spec.k.out.golden

```diff
@@ -0,0 +1,27 @@
+  <T>
+    <k>
+      DotVar2
+    </k>
+    <state>
+      n |-> 0
+      sum |-> S +Int N +Int N *Int ( N +Int -1 ) /Int 2
+    </state>
+  </T>
+#And
+  {
+    N +Int -1 >=Int 0
+  #Equals
+    true
+  }
+#And
+  {
+    N >=Int 0
+  #Equals
+    true
+  }
+#And
+  {
+    false
+  #Equals
+    N <=Int 0
+  }
```

Comment:
Two of the configurations are `\bottom`, and the other two configurations are identical. I'm pretty sure we detect and remove `\bottom` patterns for `OrPattern`s, but do we also remove duplicates?